include/fuzzuf/cli/fuzzer/aflplusplus/build_aflplusplus_fuzzer_from_args.hpp

```diff
@@ -283,7 +282,7 @@ std::unique_ptr<TFuzzer> BuildAFLplusplusFuzzerFromArgs(
 
     havoc_optimizer.reset(new optimizer::slopt::SloptOptimizer(
         AFLPLUSPLUS_NUM_CASE, GetMaxFile<AFLplusplusTag>(),
-        GetHavocStackPow2<AFLplusplusTag>()));
+        GetHavocStackPow2<AFLplusplusTag>() + 1));
```

Comment:
This `+1` should not be here, since the constructor of `SloptOptimizer` already takes care of it.

https://github.com/fuzzuf/fuzzuf/blob/355d98c5cf15950ae93f37f63ff196b8bb2433b6/optimizer/slopt/slopt_optimizer.cpp#L35

```suggestion
        GetHavocStackPow2<AFLplusplusTag>()));
```

Reply:
You're right...